test-data/unit/plugins/config_data.py

```diff
@@ -11,7 +11,7 @@ class ConfigDataPlugin(Plugin):
     def report_config_data(self, ctx: ReportConfigContext) -> Any:
         path = os.path.join("tmp/test.json")
         with open(path) as f:
-            data = json.load(f)
+            data: Any = json.load(f)
```

Comment:
```python
data: optype.json.Object
```
😏

Reply:
I will use optype when I have to write some json code, but for this, it's upstream stuff unfortunately